content/en/examples/controllers/frontend.yaml

```diff
@@ -11,8 +11,6 @@ spec:
   selector:
     matchLabels:
       tier: frontend
-    matchExpressions:
-      - {key: tier, operator: In, values: [frontend]}
   template:
     metadata:
       labels:
```

Comment:
Strong suggestion: Remove all the extraneous field from this ReplicaSet manifest. Remove the entire resources section and the entire env section. Also remove the ports section. The containerPort field is problematic, because people get the wrong impression about what it does. It actually does nothing; it's just informational. But people tend to think that it causes the container to listen on a particular port. It doesn't.